### ISSUE TITLE
docs: add simple example of event being emitted

### ISSUE THREAD

**Issue by refl3ction** [primus/eventemitter3#259]
I missed this while looking at this package, a very simple example emitting and listening to an event on the `usage` part would be nice.

**Comment by lpinca** [comment]
See https://github.com/primus/eventemitter3#usage. There is a link there (https://nodejs.org/api/events.html) with a lot of examples.

**Comment by refl3ction** [comment]
Let me rephrase it. What I can see when I read the README is just that:

```js
var EventEmitter = require('eventemitter3');
```

What about adding something like this? Is that the way we should be doing?
```js
var EventEmitter = require('eventemitter3');

class MyEmitter extends EventEmitter {}

const myEmitter = new MyEmitter();
myEmitter.on('event', () => {
  console.log('an event occurred!');
});
myEmitter.emit('event');
```

**Comment by lpinca** [comment]
You have that example here https://nodejs.org/api/events.html#emitteroneventname-listener. It is only 2 click away.

**Comment by lpinca** [comment]
Yes the link is correct, try to open in different browser (it also does not work for me on Chrome).

**Comment by refl3ction** [comment]
Yep that was it, opening it on Safari worked. So my question was more about how the maintainers recommend the users to use it. In most open source projects there is are examples on the README or an `examples` directory containing some common use case, and I think that helps users to get started. This issue can be closed.

**Comment by lpinca** [comment]
Yes, I understand but this module works like the Node.js `EventEmitter` so there is no reason to duplicate examples. This is documented.

> For the API documentation, please follow the official Node.js documentation: http://nodejs.org/api/events.html

**Comment by lpinca** [comment]
I'm closing this as answered.
